src/relations.jl

```diff
@@ -911,13 +949,13 @@ function saturation_vapor_pressure(
     cp_l::FT = TP.cp_l(param_set)
     cp_i::FT = TP.cp_i(param_set)
     # get phase partitioning
-    liquid_frac = liquid_fraction(param_set, T, phase_type, q)
-    ice_frac = 1 - liquid_frac
+    λ = liquid_fraction(param_set, T, phase_type, q)
+    ice_frac = 1 - λ
 
     # effective latent heat at T_0 and effective difference in isobaric specific
     # heats of the mixture
-    LH_0 = liquid_frac * LH_v0 + ice_frac * LH_s0
-    Δcp = liquid_frac * (cp_v - cp_l) + ice_frac * (cp_v - cp_i)
+    LH_0 = λ * LH_v0 + ice_frac * LH_s0
```

Comment:
Is there an advantage of having `ice_frac` here instead of `(1 - λ)`?

Reply:
Not really, we can inline it 🙂